applications/MedApplication/tests/test_med_model_part_io.py

```diff
@@ -110,13 +101,174 @@ def test_quadrilateral_4N_linear_mesh(self):
+            self.assertEqual(sum(exp_geoms.values()), model_part.NumberOfGeometries())
+            self.assertDictEqual(exp_geoms, get_num_geometries_by_type(model_part))
+
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeLength(model_part), 3200)
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeArea(model_part), 340000)
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeVolume(model_part), 10000000)
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeDomainSize(model_part), 10343200)
+
+            for node in model_part.Nodes:
+                self.assertTrue(0.0 <= node.X <= 500.0)
+                self.assertTrue(0.0 <= node.X0 <= 500.0)
+
+                self.assertTrue(0.0 <= node.Y <= 100.0)
+                self.assertTrue(0.0 <= node.Y0 <= 100.0)
+
+                self.assertTrue(0.0 <= node.Z <= 200.0)
+                self.assertTrue(0.0 <= node.Z0 <= 200.0)
+
+        self._execute_tests("hexahedral_20N", mp_check_fct)
+
+    def test_hexahedra_27N_biquadratic_mesh(self):
+        self.skipTest("The connectivity conversion is not yet fully implemented")
+
+        def mp_check_fct(model_part):
+            self.assertEqual(model_part.NumberOfNodes(), 36)
+            self.assertEqual(model_part.NumberOfGeometries(), 36)
+
+            # check how many geoms of each type
+            exp_geoms = {KM.Hexahedra3D27: 125, KM.Quadrilateral3D9: 150, KM.Line3D3: 60, KM.Geometry: 121}
+            self.assertEqual(sum(exp_geoms.values()), model_part.NumberOfGeometries())
+            self.assertDictEqual(exp_geoms, get_num_geometries_by_type(model_part))
+
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeLength(model_part), 3200)
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeArea(model_part), 340000)
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeVolume(model_part), 10000000)
+            self.assertAlmostEqual(KratosMed.MedTestingUtilities.ComputeDomainSize(model_part), 10343200)
+
+            for node in model_part.Nodes:
+                self.assertTrue(0.0 <= node.X <= 500.0)
+                self.assertTrue(0.0 <= node.X0 <= 500.0)
+
+                self.assertTrue(0.0 <= node.Y <= 100.0)
+                self.assertTrue(0.0 <= node.Y0 <= 100.0)
+
+                self.assertTrue(0.0 <= node.Z <= 200.0)
+                self.assertTrue(0.0 <= node.Z0 <= 200.0)
+
+        self._execute_tests("hexahedral_27N", mp_check_fct)
+
+
+def write_vtk(model_part, name):
+    # using the modeler to create elements for visualization,
+    # until the vtk-output supports geometries directly
+    modeler_parameters = KM.Parameters(
+        """{
+        "elements_list" : [{
+            "model_part_name" : "read_1",
+            "element_name" : "Element2D3N;Element2D4N;Element3D4N;Element3D8N;Element3D10N;Element3D20N;Element3D27N"
```

Comment:
I know it's is unrelated to this PR, but why isn't this a list of strings instead of a `;` delimited one??

Reply:
@rubenzorrilla